core/src/main/scala/com/nvidia/spark/rapids/tool/ClusterConfigurationStrategy.scala

```diff
@@ -265,12 +265,21 @@ class EventLogBasedStrategy(
       throw new IllegalArgumentException("Cluster information from event log must be defined"))
   }
 
-  // For onprem or cases where a matching CSP instance type is unavailable,
-  // Returns the memory per node
+  // scalastyle:off line.size.limit
+  /**
+   * For onprem or cases where a matching CSP instance type is unavailable,
+   * this method returns the memory per node.
+   *
+   * Reference:
+   * https://github.com/apache/spark/blob/branch-3.5/core/src/main/scala/org/apache/spark/resource/ResourceProfile.scala#L536
```

Comment:
nit: unnecessary reference the line code in spark. We are not copying the code.
Lets reference the [3.5.5 documentation ](https://spark.apache.org/docs/3.5.5/configuration.html#:~:text=spark.executor.memoryOverhead,pyspark.memory.) saying that this is a Spark3.x+ behavior.